manifests/c/Cryptomator/Cryptomator/1.7.0/Cryptomator.Cryptomator.locale.en-US.yaml

```
@@ -0,0 +1,43 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Cryptomator.Cryptomator
+PackageVersion: 1.7.0

```

```suggestion
PackageVersion: 1.7.0.4333
```
Must match Control Panel / Apps&Features